### PR TITLE
Implement support for Terraform v0.12

### PR DESCRIPTION
**Pull request by aaron-lane**
Fixes #26

### CODE REVIEW

**Comment by aaron-lane** [comment]
This pull request is for the 0.12 staging branch. The various changes were already merged from separate pull requests.

File: README.md [hunk 1]
```diff
@@ -217,9 +216,9 @@ The project has the following folders and files:
 
 ### Terraform plugins
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) 1.20.X
-- [terraform-provider-google-beta](https://github.com/terraform-providers/terraform-provider-google-beta) 1.20.X
+- [Terraform](https://www.terraform.io/downloads.html) 0.12
```
Comment (Jberlinsky) [inline README.md]: Let's add a note about the last 0.11.x-compatible version, like https://github.com/terraform-google-modules/terraform-google-network/pull/55

Comment (aaron-lane) [reply README.md]: Fixed in 639a3b7.

**Comment by aaron-lane** [comment]
Reviewed by @jeffmccune

**Pull request merged**